README.md

```diff
@@ -2,55 +2,103 @@
 
 [![Travis Build Status](https://travis-ci.org/oktal/pistache.svg?branch=master)](https://travis-ci.org/oktal/pistache)
 
-Pistache is a modern and elegant HTTP and REST framework for C++.
+Pistache is a modern and elegant HTTP and REST framework for C++. It is entirely written in pure-C++11 and provides a clear and pleasant API. 
 
-It is entirely written in pure-C++11 and provides a clear and pleasant API
+# Documentation
 
-Full documentation is located at [http://pistache.io](http://pistache.io).
+We are still looking for a volunteer to document fully the API. In the mean time, partial documentation is available at [http://pistache.io](http://pistache.io). If you are interested in helping with this, please open an issue ticket.
 
 # Contributing
 
-Pistache is an open-source project and will always stay open-source.  Contributors are welcome!
+Pistache is [free software](https://www.gnu.org/philosophy/free-sw.en.html). It will always stay free. Contributors are welcome!
 
-Pistache was created by Mathieu Stefani, who can be reached via [cpplang Slack channel](https://cpplang.now.sh/). Drop a private message to `@octal` and he will invite you to the channel dedicated to Pistache.
+Pistache was originally created by Mathieu Stefani, but he is no longer actively maintaining Pistache. A team of volunteers has taken over. 
 
-For those that prefer IRC over Slack, the rag-tag crew of maintainers are on #pistache on freenode.
+For those that prefer IRC over Slack, the rag-tag crew of maintainers idlie in `#pistache` on Freenode. Please come and join us!
```

Comment:
idle (spelling).

I've not been there in months.  Is it active?

Reply:
Hah ;) We've been missing you. We've been in there for ages. And fixed typo.